docs/calculations.mdx

```diff
@@ -338,7 +338,7 @@ image_sizes = creative_image_sizes ? ad_format.image_sizes
 image_bytes = SUM(image_sizes[n].width x image_sizes[n].height) x 3 / default_image_compression_ratio
 
 audio_duration = creative_audio_duration_seconds ?? ad_format.audio_duration_seconds
-audio_bytes = audio_duration_seconds x default_audio_bitrate_kbps[device_type]
```

Comment:
@bokelley Should this be `* impressions`

Reply:
correct, this should be * imps